services/src/main/java/org/fao/geonet/api/records/MetadataApi.java

```diff
@@ -173,15 +171,15 @@ public String getRecord(
             || accept.contains("application/pdf")) {
             return "forward:" + (metadataUuid + "/formatters/" + defaultFormatter);
         } else if (accept.contains(MediaType.APPLICATION_XML_VALUE)
-                || accept.contains(MediaType.APPLICATION_JSON_VALUE)) {
+            || accept.contains(MediaType.APPLICATION_JSON_VALUE)) {
             return "forward:" + (metadataUuid + "/formatters/xml");
         } else if (accept.contains("application/zip")
-                || accept.contains(MEF_V1_ACCEPT_TYPE)
-                || accept.contains(MEF_V2_ACCEPT_TYPE)) {
+            || accept.contains(MEF_V1_ACCEPT_TYPE)
+            || accept.contains(MEF_V2_ACCEPT_TYPE)) {
             return "forward:" + (metadataUuid + "/formatters/zip");
         } else {
             // FIXME this else is never reached because any of the accepted medias match one of the previous if conditions.
```

Comment:
Then this FIXME is not a FIXME anymore, right? :)

Reply:
Yes, you are right. I'm removing it.